addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -244,35 +244,34 @@ func updateResources(nannycfg *nannyconfig.NannyConfiguration, baseStorage strin
 	// Monitor only the resources specified.
 	if nannycfg.BaseCPU != nannyconfig.NoValue {
 		resources = append(resources, nanny.Resource{
-			Base:         resource.MustParse(nannycfg.BaseCPU),
-			ExtraPerNode: resource.MustParse(nannycfg.CPUPerNode),
-			Name:         "cpu",
+			Base:             resource.MustParse(nannycfg.BaseCPU),
+			ExtraPerResource: resource.MustParse(nannycfg.CPUPerNode),
+			Name:             "cpu",
 		})
 	}
 
 	if nannycfg.BaseMemory != nannyconfig.NoValue {
 		resources = append(resources, nanny.Resource{
-			Base:         resource.MustParse(nannycfg.BaseMemory),
-			ExtraPerNode: resource.MustParse(nannycfg.MemoryPerNode),
-			Name:         "memory",
+			Base:             resource.MustParse(nannycfg.BaseMemory),
+			ExtraPerResource: resource.MustParse(nannycfg.MemoryPerNode),
```

Comment:
Not sure what this change means. The MemoryPerNode is still used. Is this correct?

Reply:
ExtraPerNode got renamed to ExtraPerResource. I noticed this when running the tests.